### PR TITLE
use REGISTRIES env variable

### PR DESCRIPTION
**Pull request by karmab**
- allow to set REGISTRIES env variable to create registries.conf with insecure registries

### CODE REVIEW

File: assets/files/etc/containers/registries.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[root@u08-master-0 containers]# cat registries.conf
```
Comment (e-minguez) [inline assets/files/etc/containers/registries.conf]: :rocket:

**Comment by russellb** [comment]
Can you add some more explanation about this?  What is it?  Why is it needed?  Is there an openshift-kni/install-scripts issue that describes the need in more detail?

**Comment by karmab** [comment]
the need is for people wanting to use their own insecure registries during the install

**Comment by mcornea** [comment]
A good example where this is needed is for testing pre-release CNV 2.1 which uses images from an internal insecure registry.

**Comment by russellb** [comment]
so this can be done post-install then?  and it's needed for CNV testing?  In that case, I wouldn't put it in dev-scripts.

There is an issue for this against install-scripts though: https://github.com/openshift-kni/install-scripts/issues/12

**Comment by karmab** [comment]
> so this can be done post-install then? and it's needed for CNV testing? In that case, I wouldn't put it in dev-scripts.
> 
> There is an issue for this against install-scripts though: [openshift-kni/install-scripts#12](https://github.com/openshift-kni/install-scripts/issues/12)

so, it s useful for cnv but not only, it will probably be useful when we test ocs and would be useful for anyone wiling to deploy using a custom internal registry

**Comment by russellb** [comment]
I still see this as out of scope for dev-scripts, which is aimed at a dev env for baremetal infrastructure platform support.  Feel free to discuss this further as an add-on for install-scripts.

**Pull request closed without merging**